Automattic/wp-calypso
# Partner Portal: Add license list filter counts data layer

Blocked by:
- ~#50141~
- ~D57195-code~

Context:
* Project thread: pbtFPC-Um-p2
* i4 designs: pbtFPC-103-p2

#### Changes proposed in this Pull Request

* Add live API-driven license listing filter counts to the Partner Portal.

#### Testing instructions

* If you do not have a partner key, please contact Infinity for one or you will be unable to view the UI.
* Checkout PR locally, run `yarn && yarn start-jetpack-cloud`.
* Visit http://jetpack.cloud.localhost:3000/partner-portal and select your partner key, if prompted.
* You should be presented with a list of all of your licenses, if any. Test that all the filter counts displayed are correct based on your licenses.

![Screenshot 2021-02-17 at 18 29 03](https://user-images.githubusercontent.com/22746396/108234961-04644500-714e-11eb-849b-f2ac8c4f9f8b.png)
